Topics/Development_Process/Conda/Managing_channels.md

```diff
@@ -0,0 +1,31 @@
+
+## Managing Channels
```

Comment:
Maybe you can change it to a bigger font using only one '#', so it consists with pull request #600

Reply:
Edited.